packages/docs/site/docs/main/guides/for-theme-developers.md

```diff
@@ -4,4 +4,225 @@ slug: /for-theme-developers
+
+:::
+
+## Launching a Playground instance with a theme
+
+### Themes in the WordPress themes directory
+
+With WordPress Playground, you can quickly launch a WordPress installation using any theme available in the [WordPress Themes Directory](https://wordpress.org/themes/). Simply pass the `theme` [query parameter](/developers/apis/query-api) to the [Playground URL](https://playground.wordpress.net) like this: https://playground.wordpress.net/?theme=disco.
+
+### Themes in a GitHub repository
+
+You can also load a theme stored in a GitHub repository with a custom `blueprint.json` passed to the Playground instance.
+
+With the `themeZipFile` property of the [`installTheme` blueprint step](/blueprints/steps#InstallThemeStep), you can define a [`url` resource](/blueprints/steps/resources#urlreference) that points to the location of the `.zip` file containing the theme you want to load in the Playground instance.
+
+To avoid CORS issues, the Playground project provides a [GitHub proxy](https://playground.wordpress.net/proxy) that allows you to generate a `.zip` from a repository (or even a folder inside a repo) containing your or theme.
+
+:::tip
+[GitHub proxy](https://playground.wordpress.net/proxy) is an incredibly useful tool to load themes from GitHub repositories as it allows you to load a theme from a specific branch, a specific directory, a specific commit or a specific PR.
+:::
+
+For example, the following `blueprint.json` installs a theme from a GitHub repository leveraging the https://github-proxy.com tool:
+
+```json
+{
+	"steps": [
+		{
+			"step": "installTheme",
+			"themeZipFile": {
+				"resource": "url",
+				"url": "https://github-proxy.com/proxy/?repo=Automattic/themes&branch=trunk&directory=assembler"
+			},
+			"options": {
+				"activate": true
+			}
+		}
+	]
+}
+```
+
+A blueprint like the one above could be passed to a Playground instance [in several ways](/blueprints/using-blueprints).
+
+## Setting up a demo theme with Blueprints
+
+When providing a link to a WordPress Playground instance with a specific theme activated, you may also want to customize the initial setup for that theme.
+
+By providing a [blueprint](/blueprints/getting-started) to Playground you'll be able to load, activate, and configure a theme. Through [`steps`](/blueprints/steps) in the blueprint, you can configure an initial setup of your theme in the Playground instance.
+
+### `resetData`
+
+With [`resetData`](/blueprints/steps#resetData) step, you can remove the default content of a WordPress installation in order to import your own content.
+
+```json
+"steps": [
+	...,
+	{
+		"step": "resetData"
+	},
+	...
+]
+```
+
+### `writeFile`
+
+With [`writeFile`](/blueprints/steps#resetData) step, you can write data to a file at a specified path. You may want to use this step to write custom PHP code in a PHP file inside the `mu-plugins` of the Playground WordPress instance, so the code is executed automatically when the instance is loaded.
```

Comment:
Add "the" after "With":

With the writeFile step...

Reply:
I think you also mean "`mu-plugins` **directory**". It's not 100% clear from the text.